index.html

```diff
@@ -3,17 +3,117 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <title>Great Idea - Responsive I</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <title>Great Idea!</title>
 
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="css\index.css">
 
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
 
-<body>
-<!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
-</body>
-</html>
\ No newline at end of file
+  <body>
+   <div class="container">
+
+    
+        <nav class="navigation">
+        <div class="navi">
+          <a href="services.html">Services</a>
+          <a href="#">Product</a>
+          <a href="#">Vision</a>
+          <a href="#">Features</a>
+          <a href="#">About</a>
+          <a href="#">Contact</a>
+        </div>
+          <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
```

Comment:
```suggestion
          <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
```
Also consider adding a title property:
```suggestion
          <img class="logo" src="img/logo.png" alt="Great Idea! Company logo." title="image of great idea logo">
```